crates/parser/src/orca/instruction_parser.rs

```diff
@@ -0,0 +1,139 @@
+                    SwapAccounts {
+                        token_program: ix.accounts[0],
+                        token_authority: ix.accounts[1],
+                        whirlpool: ix.accounts[2],
+                        token_owner_account_a: ix.accounts[3],
+                        token_vault_a: ix.accounts[4],
+                        token_owner_account_b: ix.accounts[5],
+                        token_vault_b: ix.accounts[6],
+                        tick_array0: ix.accounts[7],
+                        tick_array1: ix.accounts[8],
+                        tick_array2: ix.accounts[9],
+                        oracle: ix.accounts[10],
+                    },
+                    SwapIxData {
+                        a_to_b: swap_ix_data.a_to_b,
+                        amount: swap_ix_data.amount,
+                        other_amount_threshold: swap_ix_data.other_amount_threshold,
+                        sqrt_price_limit: swap_ix_data.sqrt_price_limit,
+                        amount_specified_is_input: swap_ix_data.amount_specified_is_input,
+                    },
+                ))
+            },
+            SWAP_V2_IX_DISC => {
+                check_min_accounts_req(accounts_len, 15)?;
+                let swap_ix_v2_data: SwapV2IxData =
+                    BorshDeserialize::deserialize(&mut ix_data).unwrap();
+                Ok(OrcaProgramIx::SwapV2(
+                    SwapV2Accounts {
+                        token_program_a: ix.accounts[0],
+                        token_program_b: ix.accounts[1],
+                        memo_program: ix.accounts[2],
+                        token_authority: ix.accounts[3],
+                        whirlpool: ix.accounts[4],
+                        token_mint_a: ix.accounts[5],
+                        token_mint_b: ix.accounts[6],
+                        token_owner_account_a: ix.accounts[7],
+                        token_vault_a: ix.accounts[8],
+                        token_owner_account_b: ix.accounts[9],
+                        token_vault_b: ix.accounts[10],
+                        tick_array0: ix.accounts[11],
+                        tick_array1: ix.accounts[12],
+                        tick_array2: ix.accounts[13],
+                        oracle: ix.accounts[14],
+                    },
+                    SwapV2IxData {
+                        a_to_b: swap_ix_v2_data.a_to_b,
+                        amount: swap_ix_v2_data.amount,
+                        other_amount_threshold: swap_ix_v2_data.other_amount_threshold,
+                        sqrt_price_limit: swap_ix_v2_data.sqrt_price_limit,
+                        amount_specified_is_input: swap_ix_v2_data.amount_specified_is_input,
+                    },
+                ))
+            },
+            _ => Err(ParseError::from("Unknown instruction")),
+        }
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use yellowstone_vixen_mock::{run_ix_parse, tx_fixture, FixtureData};
```

Comment:
```suggestion
    use yellowstone_vixen_mock::tx_fixture;
```

Just need to expose tx_fixture now